frontend/src/components/form/BookingsSearch.tsx

```diff
@@ -55,16 +68,30 @@ export default function Search({
           <Form onSubmit={handleSubmit}>
             <GridRow container spacing={4}>
               <Grid item xs={6} md={3}>
-                <StyledFormControl>
-                  <StyledLabel htmlFor="locationName">
-                    Location Name
-                  </StyledLabel>
-                  <AutocompleteInput
-                    fieldName="locationName"
-                    options={locations}
-                  />
-                  <ErrorMessage name="locationName" />
-                </StyledFormControl>
+                <ACField
+                  name="locationId"
+                  label="Registry Location"
+                  options={locations}
+                  getOptionLabel={(option) => option.name}
+                  defaultValue={userLocation}
+                  onChange={(form) => (event, value) =>
+                    form.setFieldValue('locationId', value?.id)}
+                />
+              </Grid>
+              <Grid item xs={6}>
+                <StyledLabel htmlFor="save">Save Location</StyledLabel>
```

Comment:
feels like kind of redundant ui to have save location and save on the button, was this mocked up this way?